src/codeu/chat/RelayMain.java

```diff
@@ -44,22 +49,26 @@ public static void main(String[] args) {
 
     try (final ConnectionSource source = ServerConnectionSource.forPort(myPort)) {
 
+      final Server relay = new Server(1024, 16);
```

Comment:
either use constants or inline comments to improve readability by specifying what these values mean.